challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  	#implement your solution here
+	input.map { |creature| creature.capitalize }
```

Comment:
Cool. Check out these links and refactor your code for this challenge.

https://stackoverflow.com/questions/14881125/what-does-to-proc-method-mean
http://www.brianstorti.com/understanding-ruby-idiom-map-with-symbol/